### ISSUE TITLE
DMOS asteroid science not transmitting

### ISSUE THREAD

**Issue by GarwelGarwel** [Kerbalism/Kerbalism#885]
DMOS (DMagic Orbital Science) experiments, at least Laser Scan, doesn't work when attached to an asteroid.

The experiment result window shows up correctly, but the data is not collected by Kerbalism (i.e. Data tab remains empty and nothing is transmitted).

Here is what the log shows:
`[Kerbalism] ScienceDB.GetSubjectDataFromStockId Could not parse the SubjectData from subjectId 'dmlaserblastscan@AsteroidSrfLandedCarbonaceous14393197' : the body 'Asteroid' doesn't exist`
![image](https://github.com/Kerbalism/Kerbalism/assets/15203189/49a1772a-f53a-4f10-9533-1d6bafeabee1)

**Comment by gotmachine** [comment]
Yeah, it's known issue that would require additional support code, see : https://github.com/Kerbalism/Kerbalism/blob/af170bede7d857bba87ba70df8f295a5ec7def36/src/Kerbalism/Science/ScienceDB.cs#L697-L699
